src/main/java/com/iopipe/IOpipeExecution.java

```diff
@@ -351,5 +370,333 @@ public final ThreadGroup threadGroup()
 	{
 		return this.threadgroup;
 	}
+
+	/**
+	 * Builds the request which is sent to the remote service.
```

Comment:
I have to look at this in full context but ... this looks like a duplicate add? I see the PR notes about removing buildRequest() so shouldn't there be a deletion here?

Reply:
Oh, I meant from a user perspective the method is now gone. The code there is still useful and I just move it to where it is better located and make it hidden so the user cannot access it (they have no need to anyway).

Reply:
Ah got it, I was looking for a deletion, and the whole file was removed 👍 Got it